docker/dbt/Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM alpine:latest
```

Comment:
In some CI/CD stuff I've been working on for the core `dbt` package, I've been using `FROM python:3.8.1-slim-buster`, which is based on Debian Buster - is that viable here? It would be nice to have consistency across all our docker files.

I figured size would be of interest, so I did the change locally, I got this:
FROM alpine:latest -> 533MB
FROM python:3.8.1-slim-buster -> 711MB

I know I'm willing to pay a couple hundred mb - do you think that's reasonable?

(my diff, for reference):
```diff
-FROM alpine:latest
+FROM python:3.8.1-slim-buster

 ENV DBT_VERSION "0.15.2"
 ENV DBT_DIR /opt/dbt-presto

-RUN apk add python3 python3-dev postgresql-dev build-base libffi-dev git bash
+RUN apt-get update && \
+                apt-get dist-upgrade -y && \
+                apt-get install -y  --no-install-recommends \
+                    netcat curl git ssh  software-properties-common \
+                    make build-essential ca-certificates libpq-dev && \
+                apt-get clean && rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/*
```

Reply:
Sure, it makes sense to derive all docker containers from the same base image.

Changed: https://github.com/fishtown-analytics/dbt-presto/pull/10/commits/9ffc16fb447b98ce611d6bd4faecbd8057335717